src/Form/EntityEmbedDialog.php

```diff
@@ -187,6 +254,76 @@ public function buildSelectStep(array &$form, FormStateInterface $form_state) {
         'callback' => '::submitSelectStep',
         'event' => 'click',
       ),
+      '#attributes' => [
+        'class' => [
+          'js-button-next',
+        ],
+      ],
+    );
+
+    return $form;
+  }
+
+  /**
+   * Form constructor for the entity review step.
+   *
+   * @param array $form
+   *   An associative array containing the structure of the form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The current state of the form.
+   *
+   * @return array
+   *   The form structure.
+   */
+  public function buildReviewStep(array &$form, FormStateInterface $form_state) {
+    $entity_element = $form_state->get('entity_element');
+
+    /** @var \Drupal\embed\EmbedButtonInterface $embed_button */
+    $entity = $form_state->get('entity');
+
+    $form['attributes']['data-entity-type'] = array(
+      '#type' => 'value',
+      '#value' => $entity_element['data-entity-type'],
+    );
+
+    $form['attributes']['data-entity-uuid'] = array(
```

Comment:
We are also passing data-entity-id between other steps. It is deprecated but we should probably still do it since we do it everywhere else.